src/functions/proccheck.sh

```diff
@@ -117,7 +117,7 @@ proccheck() {
     fi
   fi
 
-  #check for forifty source support
```

Comment:
make build needs run before checking in.   

the `hack/enable-git-hooks.sh` script will enable the pre-commit hooks which will prevent committing without the render and do it for you so then its just adding it.